src/payment/strategies/adyenv2/adyenv2-payment-strategy.ts

```diff
@@ -315,4 +325,15 @@ export default class AdyenV2PaymentStrategy implements PaymentStrategy {
     private _updateAdyenComponentState(newState: AdyenCardState) {
         this._adyenComponentState = newState;
     }
+
+    private _getBrowserInfo(): BrowserInfoRequest {
+        return {
+            language: navigator.language || navigator.languages[0],
+            color_depth: screen.colorDepth,
+            screen_height: screen.height,
+            screen_width: screen.width,
+            time_zone_offset: new Date().getTimezoneOffset().toString(),
+            java_enabled: navigator.javaEnabled(),
```

Comment:
It seems this method is not widely supported (https://developer.mozilla.org/en-US/docs/Web/API/NavigatorPlugins/javaEnabled). Because this is a method, it'd throw an error if it's undefined. So I think you should do a check before calling it - i.e.: `typeof navigator === 'function' ? navigator.javaEnabled() : undefined` (or `false` or any other default value you think it's appropriate).

Reply:
Ok I'll write this: `typeof navigator.javaEnabled === 'function' ? navigator.javaEnabled() : false` because if I only check navigator, then `navigator.javaEnabled()` throw me an error that it does not exists in type `never`